supply_water.md

```diff
@@ -0,0 +1,112 @@
+| Desalination cost | Water basin and year | $$km^3$$ | [Exogenous](inputs_supply.html) |
+
+
+## Description
+
+Three distinct sources of fresh water are modeled, renewable water (surface and ground), non-renewable groundwater, and desalinated water. Renewable water is water that is replenished naturally by surface runoff and subsurface infiltration and release. Non-renewable groundwater is fossil groundwater resources or groundwater where abstraction exceeds recharge. Renewable water and non-renewable ground water are separately modeled for each basin. Desalinated water is available as an additional source of freshwater within each basin and for alternative end-use demands for water.
+
+### Renewable water
+
+Renewable water supplies are based on runoff estimates from Xanthos, a detailed global hydrology model ([Liu et al. 2018](#liu2018); [Vernon et al. 2019](#vernon2019)). Xanthos accounts for surface and subsurface processes to compute runoff at 0.5° grid resolution. Global climate datasets are utilized in conjunction with Xanthos to determine historical annual average runoff aggregated for each basin ([Liu et al. 2018](#liu2018); [Turner et al. 2019b](#turner2019b); [Vernon et al. 2019](#vernon2019)). Of the total basin runoff, water available or accessible for human use takes into consideration requirements for ecosystem services, inaccessibility due to rapid flow and remote locations, and capacity of reservoir storage. Accessible fractions of total runoff vary across basins. Renewable water volumes up to the accessible fraction is available at nominal or low cost. Additional renewable water beyond the accessible fraction is available at a sharply higher cost to ensure availability of water for ecosystem services and to reflect capital investments necessary for reservoir expansion.
+
+### Non-renewable groundwater
+
+Non-renewable groundwater is also represented at the basin-scale so that it is a supplemental source to basin renewable water. Non-renewable groundwater is modeled as a graded depletable resource with a fixed amount of total groundwater availability. Detailed global assessments of porosity, aquifer thickness, permeability, and water depth are utilized to establish the total volume of water available at the grid-scale. Basin level estimates of environmentally exploitable groundwater are aggregated from grid-scale data. A groundwater supply curve representing the relationship between exploitable groundwater and cost of extraction is constructed from a physics-based extraction cost model. Capital and operating costs that include well installation and maintenance costs as a function of depth and geological complexity, and energy inputs and costs required for pumping are included for a rigorous estimate of the relationship between groundwater volume and extraction cost. ([Yonkofski et al. 2019](#yonkofski2019); [Turner et al. 2019a](#turner2019a); [Kim et al. 2016](#kim2016)).
```

Comment:
This one is tough. This is how the process is completed, however it is almost solely based upon the Yonkofski 2019 paper on superwell which was never published and is now no longer in review. I think it should be left as is, as this is the process that Sean used in the 2019 paper cited (which has been published). Only thing would be to remove the Yonkofski citation, unless we want to have more hand waving here which I am not a fan of.